akka-cluster-sharding-typed/src/test/java/jdocs/akka/cluster/sharding/typed/AccountExampleWithCommandHandlersInState.java

```diff
@@ -1,297 +0,0 @@
-/*
- * Copyright (C) 2018-2019 Lightbend Inc. <https://www.lightbend.com>
- */
-
-package jdocs.akka.cluster.sharding.typed;
-
-import akka.actor.typed.ActorRef;
-import akka.actor.typed.Behavior;
-import akka.actor.typed.javadsl.Behaviors;
-import akka.cluster.sharding.typed.javadsl.EntityTypeKey;
-import akka.cluster.sharding.typed.javadsl.EventSourcedEntityWithEnforcedReplies;
-import akka.persistence.typed.ExpectingReply;
-import akka.persistence.typed.PersistenceId;
-import akka.persistence.typed.javadsl.CommandHandlerWithReply;
-import akka.persistence.typed.javadsl.CommandHandlerWithReplyBuilder;
-import akka.persistence.typed.javadsl.EventHandler;
-import akka.persistence.typed.javadsl.EventHandlerBuilder;
-import akka.persistence.typed.javadsl.ReplyEffect;
-
-import java.math.BigDecimal;
-
-/**
- * Bank account example illustrating: - different state classes representing the lifecycle of the
- * account - event handlers that delegate to methods in the state classes - command handlers that
- * delegate to methods in the state classes - replies of various types, using ExpectingReply and
- * EventSourcedBehaviorWithEnforcedReplies
- */
-public interface AccountExampleWithCommandHandlersInState {
```

Comment:
had to remove AccountExampleWithCommandHandlersInState because not possible to serialize State when it's not static
  * Effect() is factory in EventSourcedBehavior  class